oss-internship-2020/pffft/README.md

```diff
@@ -0,0 +1,86 @@
+# Sandboxing PFFFT library
+
+Build System: CMake  
+OS: Linux
+
+### Check out the PFFFT library & CMake set up
+`git clone https://github.com/doinachiroiu/sandboxed-api/tree/master/oss-internship-2020/pffft`
+
```

Comment:
Remove this, users will need to checkout the root repository. Also, you cannot `git clone` a sub-directory of a Git repo.